ios/RNNativePlayground/ViewController.m

```diff
@@ -22,67 +26,63 @@ - (instancetype)initWithCoder:(NSCoder *)coder {
      * RNKiwiViewController does exactly the same thing for you behind the scenes if you use
      * the former approach.
      */
-    [[RNKiwiSharedBridge sharedInstance] initBridgeWithOptions:self];
+    [[RNKiwiSharedBridge sharedInstance] initBridgeWithOptions:[[RNOldHotelsOptions alloc] init]];
```

Comment:
This is confusing me. Why it uses `RNOldHotelsOptions` ? Isn't this shared between old and new?